scripts/mkcloud

```diff
@@ -808,12 +807,12 @@ function crowbarupgrade_5plus
 {
     if iscloudver 6plus ; then
         onadmin upgrade_prechecks
+        export cloudsource=$upgrade_cloudsource
         onadmin prepare_crowbar_upgrade
         onadmin prepare_cloudupgrade_repos_6_to_7
         onadmin upgrade_admin_backup
         onadmin upgrade_admin_repocheck
         onadmin upgrade_admin_server
-        export cloudsource=$upgrade_cloudsource
```

Comment:
I do not think this is needed. `$cloudsource` is not used by the previous steps (and if it is, the old one should be). It's after the admin server upgrade we need to tell mkcloud to use new value.